apps/studio/src/features/editing-experience/components/form-builder/renderers/layouts/JsonFormsVerticalLayout.tsx

```diff
@@ -104,7 +105,7 @@ export function JsonFormsVerticalLayoutRenderer({
   const newElements = getUiSchemaWithGroup(schema, elements)
 
   return (
-    <>
+    <Box w="100%">
```

Comment:
bugfix so item will be full width, not sure if there are any downstream impact (ie, across our other jsonforms component) but based off just clicking aorund the UI, it seems ok